### PR TITLE
#166901840 converting to es6

### PR DESCRIPTION
**Pull request by shaluchandwani**
#### What does this PR do?
- Removed unwanted code
- changed code to es6 format

#### Description of Task to be completed?
Code base changed to es6 format

#### How should this be manually tested?
Using npm run start

#### Any background context you want to provide?
na
#### What are the relevant pivotal tracker stories?
na
#### Screenshots (if appropriate)
na
#### Questions:
na

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,98 +1,23 @@
...
-
-// finally, let's start our server...
-const server = app.listen(process.env.PORT || 3000, function() {
+const server = app.listen(process.env.PORT || 3000, () => {
     console.log("Listening on port " + server.address().port);
 });
 
-module.exports = server
\ No newline at end of file
+module.export = server;
```
Comment (Joe-Joseph) [inline index.js]: I'm not sure if this is ES5 or ES6. I thought the ES6 syntax should be export default.

Good job adding .hound.yml  it has done all the review

File: index.js [hunk 2]
```diff
@@ -1,98 +1,23 @@
...
-//             message: err.message,
-//             error: {}
-//         }
-//     });
-// });
-
-// finally, let's start our server...
-const server = app.listen(process.env.PORT || 3000, function() {
+const server = app.listen(process.env.PORT || 3000, () => {
     console.log("Listening on port " + server.address().port);
```
Comment (Joe-Joseph) [inline index.js]: Here we can use template literal to avoid that plus and it's also ES6 feature.

**Review by Joe-Joseph — COMMENTED** [review]
Thanks for doing a good job. I've tested your branch and it is running, you can take correct those comment of HoundCI, and then your work will be ready to be merged

**Comment by mnzube** [comment]
Hey Shalu, 
- Maybe you can add some little info for the background context for this PR 
- Also, this PR has a PT story has an ID.
- I think you could use N/A instead of 'na'

**Pull request merged**